### PR TITLE
Challenge & improve UI edge cases for form elements (issue/3447)

### LINKED ISSUE #3447: Challenge & improve UI edge cases for form elements
Scope
---
In case form elements (e.g. select, textfield, etc.) are used with very limited space they might not be handled ideally from a UI perspective. Therefore, we want to challenge and improve UI edge case for those with limited horizontal space.

Notes  
---
Community Thread: https://porschedev.slack.com/archives/CT7AVHTTQ/p1723124568510879

Acceptance criteria
---
- [ ] Form Elements are able to handle very limited space from a UI perspective
- [ ] Padding calculation for the unit of `p-text-field-wrapper` works reliable

Subtasks
---
- [ ] Task

### PR DESCRIPTION
**Pull request by o-hook**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/3447

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves #3447

### CODE REVIEW

File: packages/components/src/components/text-field-wrapper/text-field-wrapper-styles.ts [hunk 1]
```diff
@@ -102,6 +102,10 @@ export const getComponentCss = (
           : formElementPaddingHorizontal,
       display: 'grid',
       gap: spacingStaticXSmall,
+      // min width is needed for showing at least 1 character in very narrow containers
+      ...((hasUnitOrVisibleCounter || isSearch || isPassword || isCalendarOrTimeWithCustomIndicator) && {
+        minWidth: `calc(${formElementPaddingHorizontal} + ${borderWidthBase} * 2 + ${getCalculatedFormElementPaddingHorizontal(isSearch ? 2 : 1)} + 2rem)`,
```
Comment (henrijoss) [inline packages/components/src/components/text-field-wrapper/text-field-wrapper-styles.ts:107]: Maybe add comment explaining calculation, why `+ 2rem`?

File: packages/components/src/components/select/select/select-styles.ts [hunk 2]
```diff
@@ -70,6 +70,8 @@ export const getComponentCss = (
     root: {
       display: 'grid',
       gap: spacingStaticXSmall,
+      // min width is needed for showing at least 1 character in very narrow containers
+      minWidth: `calc(${formElementPaddingHorizontal} + ${borderWidthBase} * 2 + ${getCalculatedFormElementPaddingHorizontal(1)} + 1rem)`,
```
Comment (henrijoss) [inline packages/components/src/components/select/select/select-styles.ts:74]: Why `1rem` here?

File: packages/components-js/src/pages/multi-select.html [hunk 3]
```diff
@@ -386,6 +386,16 @@
   </p-multi-select>
 </div>
 
+<div class="playground light" title="should render with ellipsis and min-width in a smaller container">
+  <div style="width: 60px">
```
Comment (henrijoss) [inline packages/components-js/src/pages/multi-select.html:390]: Still `60px`?

Comment (o-hook) [reply packages/components-js/src/pages/multi-select.html:390]: Yes, because the component is wider.

**Review by henrijoss — APPROVED** [review]

**Pull request merged**